src/App.jsx

```diff
@@ -1,33 +1,13 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { BrowserRouter } from 'react-router-dom'
+import navbar from './components/navbar'
+
 
-function App() {
-  const [count, setCount] = useState(0)
 
+function App() {
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+    <><BrowserRouter>
+    {navbar()}  
```

Comment:
**[OPTIONAL]** 

Consider rendering a component using the react way `</Component />` and not as an explicit function call like this so that props can be passed easily. 

Also, it seems weird that a navbar is rendering a form and a whole page. Consider seperating the concerns like this

```
<>
  <NavBar />
  <Routes>
    <Route> Your routes </Route>
  </Routes>
</Routes>
```